### PR TITLE
Linear view mscolab bug

### LINKED ISSUE #1022: Table view not opening once Linear view is created in MSColab
Trying to create tableview after creating linearview in an MSColab Project will just raise the already existing linearview window.

### PR DESCRIPTION
**Pull request by aravindm711**
1. Tableview does not open after opening linearview
2. On closing a view main window appears instead of mscolab window

### CODE REVIEW

File: mslib/msui/mscolab.py [hunk 1]
```diff
@@ -1271,6 +1271,7 @@ def handle_view_close(self, value):
         for index, window in enumerate(self.active_windows):
             if window._id == value:
                 del self.active_windows[index]
+                self.raise_()
```
Comment (Marilyth) [inline mslib/msui/mscolab.py:1274]: I am not sure why, but this line does nothing on my end. Same for self.show(), self.activateWindow() and self.showNormal().
Maybe that is a Linux issue.
On Windows I can not even change the order of the windows. MSColab is always in front of Main and the views are always in front of MSColab.
What OS are you using for the main window to pop up after closing a window?

**Review by Marilyth — COMMENTED** [review]
Thanks. The linearview fix is also part of #1020 but it can't hurt to fix it sooner.

File: mslib/msui/mscolab.py (hunk 1 above)
Comment (aravindm711) [reply mslib/msui/mscolab.py:1274]: I'm using a mac for development. It's usually the thing with Macs, the first created window would raise to the front.

Comment (Marilyth) [reply mslib/msui/mscolab.py:1274]: Unfortunately I can't test this on mac, but I will take your word for it working the way it should.

**Review by Marilyth — APPROVED** [review]

**Pull request merged**